ci-jobs/pr-validation-build.yml

```diff
@@ -13,20 +13,20 @@ stages:
     parameters:
       name: iPhoneX_15_2
       iosVersion: 15.2
-      xcodeVersion: 13.2
+      xcodeVersion: 14.0
       deviceName: "iPhone Simulator"
       #tvosVersion: 13.4
       #tvosName: AppleTV_13_4
       #tvosDeviceName: "Apple TV"
       skipTvOs: True
-      vmImage: 'macOS-11'
+      vmImage: 'macOS-12'
```

Comment:
https://github.com/actions/runner-images/issues/6384
macos-latest still points to 11